### PR TITLE
AWS codebuild for windows: set build directory

### PR DESCRIPTION
**Pull request by kroening**
With this PR, clcache finally works: the build time is reduced from 15 minutes to 3 mins when there are no changes to any of the sources.

### CODE REVIEW

**Review by allredj — COMMENTED** [review]
Passed Diffblue compatibility checks (cbmc commit: 06dd8bd221f46d2e19851849ca8dea5d44b3016b).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/80993957

File: buildspec-windows.yml [hunk 1]
```diff
@@ -3,6 +3,13 @@ version: 0.2
 phases:
   install:
     commands:
+      - |
+        # Codebuild sets up a directory whose name changes with every build.
+        # We create a symlink with a fixed name to enable clcache to get cache hits.
+        Set-Location -Path "c:\codebuild\tmp"
+        New-Item -ItemType Junction -Name builddir -Value $env:CODEBUILD_SRC_DIR
+        Set-Location -Path "c:\codebuild\tmp\builddir"
```
Comment (tautschnig) [inline buildspec-windows.yml]: This doesn't seem to be necessary: use `$env:CLCACHE_BASEDIR = (Get-Item -Path ".\").FullName` instead.

File: buildspec-windows.yml [hunk 2]
```diff
@@ -11,7 +18,7 @@ phases:
     commands:
       - |
         $env:Path = "C:\tools\cygwin\bin;$env:Path"
-        C:\tools\cygwin\bin\bash -c "make -C src minisat2-download DOWNLOADER=wget"
+        bash -c "make -C src minisat2-download DOWNLOADER=wget"
```
Comment (tautschnig) [inline buildspec-windows.yml:14]: Ok, but should be a separate commit.

**Review by tautschnig — CHANGES_REQUESTED** [review]

**Review by allredj — COMMENTED** [review]
Passed Diffblue compatibility checks (cbmc commit: cd4a163fb82c1d11ae5485880a944b43195fe0e7).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/81022065

**Review by tautschnig — APPROVED** [review]
